src/Commands/BaseCommand.cs

```diff
@@ -43,7 +43,7 @@ protected virtual void HandleException(CommandResponse response, Exception ex)
     {
         // Don't clear arguments when handling exceptions
         response.Status = GetStatusCode(ex);
-        response.Message = GetErrorMessage(ex);
+        response.Message = GetErrorMessage(ex) + ".  See https://aka.ms/azmcp/troubleshooting for more information.";
```

Comment:
" To mitigate this issue, please refer to the troubleshooting guidelines here at "